src/examples/page-heading/with-subheading-welsh/index.njk

```diff
@@ -0,0 +1,9 @@
+---
+layout: layout-example.njk
+---
+{% from "hmrc/components/page-heading/macro.njk"  import hmrcPageHeading %}
+
+{{ hmrcPageHeading({
+  text: "Beth yw’ch enw?",
+  section: "Manylion personol"
```

Comment:
There's some visually hidden wording here which needs changing in hmrc-frontend. I've added details to the Trello board. Do we have the Welsh for 'This section is'?